test/simple/simple.status

```diff
@@ -1 +1,35 @@
 prefix simple
+
+test-http-client-timeout-event          : PASS,FLAKY
```

Comment:
I looked at recent logs and I also ran the job in the new Jenkins a few times collecting all the failures.